### PR TITLE
Add new D78,D79 geometries based on D76 with T22,T23

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
This PR is to create updated detector geometries with square (T22) and 3D_pixel (T23) trackers with recent subdetectors (D76).

The next step/PR is to clean up obsolete geometries.

#### PR validation:
Test with
> runTheMatrix.py -w upgrade -l 35434.0,35834.0 --wm init

#### if this PR is a backport please specify the original PR and why you need to backport that PR:
No backport is needed.

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by srimanob** [comment]
Please test

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 1]
```diff
@@ -1216,6 +1220,20 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C11M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
     },
+    '2026D78' : {
+        'Geom' : 'Extended2026D78',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T22',
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:1226]: @srimanob this is configured incorrectly.
Please add:
```python
 'ProcessModifier': 'PixelCPEGeneric',   # This swaps template reco CPE for generic reco CPE`
 ```
 as done here:
 
 https://github.com/cms-sw/cmssw/blob/db968ce4dce2f0fd9928d40c209fc2bbbf1ac3f7/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1130 
 
 for geometry D64.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 2]
```diff
@@ -1216,6 +1220,20 @@ def condition(self, fragment, stepList, key, hasHarvest):
...
+        'Era' : 'Phase2C11T22M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
+    },
+    '2026D79' : {
+        'Geom' : 'Extended2026D79',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T23',
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:1233]: @srimanob this is configured incorrectly.
Please add:
```python
 'ProcessModifier': 'PixelCPEGeneric',   # This swaps template reco CPE for generic reco CPE`
 ```
 as done here:
 
https://github.com/cms-sw/cmssw/blob/db968ce4dce2f0fd9928d40c209fc2bbbf1ac3f7/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1138

for geometry D65.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py (hunk 1 above)
Comment (srimanob) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:1226]: Fixed, Thanks.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py (hunk 2 above)
Comment (srimanob) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:1234]: Fixed, thanks.

**Comment by srimanob** [comment]
Please test

**Comment by srimanob** [comment]
Please test

**Comment by srimanob** [comment]
+Upgrade

**Comment by cvuosalo** [comment]
Workflow 11634.911 in the comparison tests somehow got a new random seed, which created statistical fluctuations in the results. The differences are not significant.

**Comment by cvuosalo** [comment]
+1

**Comment by chayanit** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Comment by makortel** [comment]
Seems that this PR could be causing workflow failures in CMSSW_11_3 2021-02-23-2300, see https://github.com/cms-sw/cmssw/issues/32986

**Pull request merged**